src/main/java/racinggame/domain/Car.java

```diff
@@ -0,0 +1,24 @@
+package racinggame.domain;
+
+public class Car {
+
+    private Position position;
+
+    public Car() {
+        this(new Position(0));
+    }
+
+    public Car(Position position) {
+        this.position = position;
+    }
+
+    public void run(MoveCondition condition) {
+        if(condition.isMovable()) {
```

Comment:
IntelliJ IDEA를 사용 중이시면 자동 코드 포맷팅 단축키(`Ctrl + Alt + L`)를 활용해보시는 걸 추천 드립니다.
```suggestion
        if (condition.isMovable()) {
```